pkg/cmd/modeline.go

```diff
@@ -53,7 +53,7 @@ var (
 	// file options must be considered relative to the source files they belong to
 	fileOptions = map[string]bool{
 		"resource":      true,
-		"config":        true,
+		"kube-client":   true,
```

Comment:
I think it's `kube-config`. This part also opens a question, because when running an integration file, one expects files declared in modeline to be relative to the file location, not to the cwd. So it should work the same for files declared as `config`/`resource` with `file:` prefix.

Otherwise the outcome of running an integration file depends on the current dir when you run the kamel command.

Reply:
You are right. I am checking if it would be enough to include `config` as well.